src/pages/kb/user-guide/querying/scheduling-a-query.md

```diff
@@ -6,23 +6,28 @@ title: Scheduling A Query
 slug: scheduling-a-query
 ---
 
-By default, your query won't have any scheduling but it's super easy to
-adjust!
-
-In the bottom left corner, you'll see the schedule area:
+You can use scheduled query executions to keep your public dashboards updated or to power routine Alerts. By default, your queries will not have a schedule. But this is easy to adjust. In the bottom left corner of the query editor you'll see the schedule area:
```

Comment:
```suggestion
You can use scheduled query executions to keep your dashboards updated or to power routine Alerts. By default, your queries will not have a schedule. But this is easy to adjust. In the bottom left corner of the query editor you'll see the schedule area:
```

(it's relevant for any type of dashboard)

Reply:
Fixed in c43fcf0.